src/state/base-state.ts

```diff
@@ -79,10 +61,22 @@ export abstract class BaseState<T> implements State<T> {
   /**
    * Return an action by name.
    *
-   * If the format provides a default action, the name may be omitted.
+   * If no name is given, the first action is returned. This is useful for
+   * formats that only supply 1 action, and no name.
    */
   action<TFormData = any>(name?: string): Action<TFormData> {
```

Comment:
The availability of an action is often server side computed, function of the current state, user permission, ...
Could we have a method `hasAction` allowing us to know if the action is available before calling the method `action`? Or maybe action could return an 'optional' `Action` ?

Reply:
Oh I didn't see `actions` method.
So `hasAction` would be a method allowing to look if an `action` is present by its name.

Reply:
Sounds good, will add..

Reply:
Thanks !